<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnSelectInputFile1">
        <property name="text">
         <string>Выбрать...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInputFile1">
        <property name="text">
         <string>Файл 1 не выбран</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectInputFile2">
        <property name="text">
         <string>Выбрать...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInputFile2">
        <property name="text">
         <string>Файл 2 не выбран</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableMatrix1"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableMatrix2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnExecute">
      <property name="text">
       <string>&gt; Умножить &gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSaveResult">
        <property name="text">
         <string>Сохранить результат как...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableOutputMatrix"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
